### DISCUSSION TITLE
Combine: owner and access columns

### DISCUSSION THREAD

**Discussion by shivarm** [Hyperfoil/Horreum#856] [General]
I followed our previous discusssion https://github.com/Hyperfoil/Horreum/pull/526#issuecomment-1547726857 and created a branch please have look or you are able to track my branch https://github.com/Hyperfoil/Horreum/compare/master...shivam-sharma7:Horreum:combine-columns  test it. so that i can open PR :)

This has done: #876

**Comment by shivarm** [comment]
actually you know what is reason for do it! I'm waiting for this https://github.com/Hyperfoil/Horreum/issues/787  which i think can solve my problem

**Reply by johnaohara** [reply]
@shivam-sharma7 please can you test this PR on your windows machine: https://github.com/Hyperfoil/Horreum/pull/857

This should allow you to load the sample data via the dev services UI (http://localhost:8080/q/dev-ui/extensions) instead of using the script in `infra-legacy/example-configuration.sh`

**Reply by shivarm** [reply]
@johnaohara see

![image](https://github.com/Hyperfoil/Horreum/assets/91419219/d55e4910-9a41-43e2-9f7d-96ef173ca139)

Servicess running

![image](https://github.com/Hyperfoil/Horreum/assets/91419219/f21679fd-1091-4a9e-9731-b047fd7ec758)

**Reply by shivarm** [reply]
CMD used
```
 mvn -DskipTests=true -DskipITs install
 mvn quarkus:dev
```

**Reply by shivarm** [reply]
I have tested other cmd but same error
